### ISSUE TITLE
Investigate the support for the different aria-haspop properties

### ISSUE THREAD

**Issue by terracoda** [phetsims/a11y-research#109]
The Keyboard Shortcuts button, does not read out as a a "pop-up" button, so I am not sure it is coded as one. It is technically a pop-up button. It pops-up a dialog, and I thought it should have an aria-haspopup="dialog", but this might need more testing.

See article from Level Access
https://www.levelaccess.com/expanding-the-use-of-the-aria-haspopup-property/

Issue first mentioned in ohms-law#115

**Comment by terracoda** [comment]
Adding resource:
http://www.sumandamera.com/aria-haspopup-property/

**Comment by terracoda** [comment]
The issue of a dialog using a "pop-up" button came up as a comment in an interview for the Molecules and Light sim (Dec 20, 2019). Everything worked fine, but consultant said "has pop-up" denoted a menu interaction to her - not a dialog.

We may need to check on the current level of support for the "has dialog button".
